docs/_doc/Setting-up-monitoring-on-local-cluster.md

```diff
@@ -1,57 +0,0 @@
----
-title: Setting up monitoring on local cluster
```

Comment:
@fjl And this one? Though there are traces of this in other docs, so I can see clearer replacements

Reply:
This page was way outdated. Nowadays geth just includes an ethstats client (`--ethstats`) and puppeth can be used to deploy the stats server.